src/HonzaBotner.Services.Contract/HonzaBotner.Services.Contract.csproj

```diff
@@ -1 +1,5 @@
-<Project Sdk="Microsoft.NET.Sdk" />
+<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <ProjectReference Include="..\HonzaBotner.Database\HonzaBotner.Database.csproj" />
+  </ItemGroup>
+</Project>
```

Comment:
Remove this dependency. From contract should not have any unnecessary dependencies.